certified-connectors/Oneflow/README.md

```diff
@@ -0,0 +1,26 @@
+
+# Oneflow  
+Be more effective with truly digital contracts that make work faster and life easier. Automate your contract management process with Oneflow.
+
+# Prerequisites  
+To use the Oneflow connector, you need an active Oneflow account. If you don’t have an account yet, go ahead and [create a trial account]([https://app.oneflow.com/signup](https://app.oneflow.com/signup)).
+
```

Comment:
@DmitryKleshchov Please follow the templates for readme.md provided in the repo and add required sections.

Reply:
done